addons/website_forum/controllers/main.py

```diff
@@ -86,8 +89,19 @@ def questions(self, forum, tag=None, page=1, filters='all', sorting=None, search
             domain += [('tag_ids', 'in', tag.id)]
         if filters == 'unanswered':
             domain += [('child_ids', '=', False)]
-        elif filters == 'followed':
-            domain += [('message_partner_ids', '=', request.env.user.partner_id.id)]
+        if filters == 'solved':
```

Comment:
use elif

if your enter in the first if, it become useless to test the next ones if